aries-site/src/examples/components/layer/LayerNotificationExample.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import { Button, Box, Layer, Text } from 'grommet';
+import { StatusGood, FormClose } from 'grommet-icons';
+
+export const LayerNotificationExample = () => {
+  const [open, setOpen] = useState(false);
+  const onOpen = () => setOpen(true);
+  const onClose = () => setOpen(undefined);
+  return (
+    <>
+      <Box align="start">
+        <Button label="Show me the notification" onClick={onOpen} primary />
+      </Box>
+      {open && (
+        <Layer
+          position="top"
+          modal={false}
+          margin={{ vertical: 'medium', horizontal: 'small' }}
+          onEsc={onClose}
```

Comment:
The onEsc isnt working here

Reply:
Good catch -- I believe since we applied modal={false}, then the keyboard isn't restricted to the layer and so the onClose isn't having an effect. I will investigate more, but for now I'll remove it since it's not having an effect.